index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Koshao's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="container-toolbar">
+            <a href="#" class="toolbar-title">Welcome</a>
+            <nav class="toggle">
+                <!--<ul class="nav-menu">
+                    <li class="nav-item">
+                       <a href="#" class="nav-link">Home</a>
+                    </li>
+                    <li class="nav-item">
+                       <a href="#" class="nav-link">About</a>
+                    </li>
+                    <li class="nav-item">
+                       <a href="#" class="nav-link">Contact</a>
+                    </li>
+                 </ul>-->
+
+                 <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                 </div>
+            </nav>
+        </div>
+    </header>
+
+    <section class="headline">
+        <div class="headline-name-job">
+            <h1 class="headline-name">Hey there. I'm Koshao</h1>
+            <h1 class="headline-job">I'm a software developer</h1>
```

Comment:
[OPTIONAL] I think it is not semantically appropriate to be having two main headings😉. I suggest you change `<h1 class="headline-job">I'm a software developer</h1>` to <span class="headline-job">I'm a software developer</span> and put it inside `<h1 class="headline-name">Hey there. I'm Koshao</h1>` for styling. you can then use a `<br>` tag between them so it starts on a new line.😉